benchmark/python/benchmark_e2e.py

```diff
@@ -226,9 +226,8 @@ def run_benchmark(args, batch_size, prompt_length, generation_length, max_length
     if args.verbose: print("Running warmup runs...")
     for _ in tqdm(range(args.warmup)):
         generator = og.Generator(model, params)
-        while not generator.is_done():
-            generator.compute_logits()
-            generator.generate_next_token()
+        generator.compute_logits()
```

Comment:
Why is there no more generation loop? It seems to just run once? Is this just to "process the prompt tokens" and not to generate any tokens beyond it?

Reply:
it is for warming up. ORT can be warmed up by just running the prompt